packages/@lwc/engine-core/package.json

```diff
@@ -24,6 +24,7 @@
         "types/"
     ],
     "dependencies": {
+        "@lwc/aria-reflection": "2.32.0",
```

Comment:
Had to be careful with where to put `@lwc/aria-reflection` in the `dependencies`/`devDependencies` so that the dependency wouldn't get duplicated in the `dist` output, now that it's depended on by both `@lwc/engine-core` and `@lwc/engine-dom`.